server/schema/schema.js

```diff
@@ -216,6 +217,29 @@ const RootQuery = new GraphQLObjectType({
   },
 });
 
+const Mutation = new GraphQLObjectType({
+  name: 'Mutation',
+  fields: {
+    createCohort: {
+      type: CohortType,
+      args: {
+        name: { type: GraphQLString },
+        phase: { type: GraphQLString },
+      },
+      resolve(parent, args) {
+        let { name, phase } = args;
+        name = name.toLowerCase();
+        return cohorts.addCohort({ name, phase })
+          .then((result) => {
+            return { id: result.cohort_id, name: result.cohort_name, phase: result.phase }
```

Comment:
We need to decide whether to do the mapping of snake_case to camelCase in the graphQL layer or the db models.  I think a case could be made for either, so long as the client requests include the simplified camelCase.  @magee preferences?

Reply:
@michellelockett I pushed in a change to do the mapping in db model. Please take a look.